### PR TITLE
Add missing timeseries tests and change old names

### PR DESCRIPTION
**Pull request by hazen**
The initial handful of TS tests changed names during development and a another group of tests were added.

### CODE REVIEW

File: priv/sql/163-update-timeseries.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+INSERT INTO projects_tests (project_id, test_id)
+   SELECT projects.id, newtests.id FROM projects, newtests
+    WHERE projects.name = 'riak_ts';
+
+UPDATE tests set name = 'ts_A_activate_table_pass_1' where name = 'ts_activate_table_pass_1';
+UPDATE tests set name = 'ts_A_create_table_fail_1' where name = 'ts_create_table_fail_1';
+UPDATE tests set name = 'ts_A_create_table_pass_1' where name = 'ts_create_table_pass_1';
+UPDATE tests set name = 'ts_A_put_fail_1' where name = 'ts_put_fail_1';
+UPDATE tests set name = 'ts_A_put_fail_2' where name = 'ts_put_fail_2';
+UPDATE tests set name = 'ts_A_put_fail_3' where name = 'ts_put_fail_3';
```
Comment (gordonguthrie) [inline priv/sql/163-update-timeseries.sql:31]: No ts_A_put_fail_3

Comment (hazen) [inline priv/sql/163-update-timeseries.sql:31]: Looks like @andytill just deleted it

```
commit f050c574f4e5d54ce9909586cb48921ba4269f95
Author: andytill <atill@basho.com>
Date:   Wed Sep 30 14:16:40 2015 +0000

    Deleted tests/ts_A_put_fail_3.erl which fails ont he erlang client side so no longer needs to be an integrated test.

 tests/ts_A_put_fail_3.erl |   23 -----------------------
 1 files changed, 0 insertions(+), 23 deletions(-)
```

Comment (hazen) [inline priv/sql/163-update-timeseries.sql:31]: I think I'll leave it in until we make the effort to clean up old/dead test results

**Comment by ooshlablu** [comment]
Looks good to me. If it worked in vagrant, merge it.

**Pull request merged**